clang/lib/3C/Constraints.cpp

```diff
@@ -96,8 +96,15 @@ void Constraints::editConstraintHook(Constraint *C) {
 bool Constraints::addConstraint(Constraint *C) {
   editConstraintHook(C);
 
+  // TODO:
+  //   currently the multiple names constraint reason
```

Comment:
What's the status of this TODO? By my reading of `ProgramInfo::addTypedef`, I think this isn't a problem anymore.